### PR TITLE
Bugfix/characters tab

### PR DESCRIPTION
**Pull request by jeanawhou**
Tickets: 
https://airtable.com/appfbr7vlSqIfZ9Sn/tblAmQkxs32m46J4N/viwDW5z6rnAo0iXmL/reczmXBuxlJCOtZEI?blocks=hide
https://airtable.com/appfbr7vlSqIfZ9Sn/tblAmQkxs32m46J4N/viwDW5z6rnAo0iXmL/rec2GU04R41w7ha6G?blocks=hide
https://airtable.com/appfbr7vlSqIfZ9Sn/tblAmQkxs32m46J4N/viwDW5z6rnAo0iXmL/reclJSdbyxMDX0PKi?blocks=hide

Demo

https://user-images.githubusercontent.com/19387007/188116119-f6721669-aced-48ad-ae72-f79a8d26f117.mp4

### CODE REVIEW

**Comment by cameronsutter** [comment]
The demo LGTM

File: lib/plottr_components/src/components/characters/CharacterListView.js [hunk 1]
```diff
@@ -305,6 +305,23 @@ const CharacterListViewConnector = (connector) => {
     const renderVisibleCharacters = (categoryId) => {
       if (!visibleCharactersByCategory[categoryId]) return []
 
+      if (!!attributeTabId && attributeTabId !== 'all') {
```
Comment (Quiescent) [inline lib/plottr_components/src/components/characters/CharacterListView.js]: This needs to be a selector.
If the selector that we're using doesn't have a name that implies that it has this "filtering by book" functionality, then please add another selector.

**Comment by Quiescent** [comment]
@jeanawhou changes look great.
Could you please move the logic for filtering characters out of the component?

Context: we haven't been strict enough about this in the past; it makes it difficult to maintain the code, and the mobile team needs to replicate behaviour each time we do it.

File: lib/plottr_components/src/components/characters/CharacterListView.js (hunk 1 above)
Comment (jeanawhou) [reply lib/plottr_components/src/components/characters/CharacterListView.js]: @Quiescent got it thank you! 🙇‍♀️

**Comment by jeanawhou** [comment]
> @jeanawhou changes look great. Could you please move the logic for filtering characters out of the component?
> 
> Context: we haven't been strict enough about this in the past; it makes it difficult to maintain the code, and the mobile team needs to replicate behaviour each time we do it.

@Quiescent I see got it thank you for checking 🙇‍♀️

**Comment by Quiescent** [comment]
Excellent work with that last commit, @jeanawhou!

**Review by Quiescent — APPROVED** [review]

**Pull request merged**